packages/@icon-magic/generate/test/svg-generate-test.ts

```diff
@@ -32,6 +32,7 @@ const icon = new Icon({
   sizes: [8, 16],
   resolutions: [1, 2, 3],
   outputPath: `/${FIXTURES}/out`,
+  rtlFlip: true,
```

Comment:
you should remove this from here and only add it to one test, the one where you're testing "adds class "rtl-flip" if rtlFlip is true"

Reply:
Ok, I'm creating a new test icon `iconFlip`. Thx.